README.rst

```diff
@@ -222,7 +222,7 @@ script (``php-script`` for PHP scripts, ``script`` for any shell script and
 .. code-block:: json
 
     {
-        "composer-scripts": {
+        "auto-scripts": {
```

Comment:
We should probably add the wrapping `scripts` entry to make it very clear. WDTY?

Reply:
Sorry for the delay, done.